sky/serve/constants.py

```diff
@@ -21,6 +21,17 @@
 # interval.
 LB_CONTROLLER_SYNC_INTERVAL_SECONDS = 20
 
+# The maximum retry times for load balancer for each request. After changing to
+# proxy implementation, we do retry for failed requests.
+# TODO(tian): Expose this option to users in yaml file.
+LB_MAX_RETRY = 3
+
+# The timeout in seconds for load balancer to wait for a response from replica.
+# Large LLMs like Llama2-70b is able to process the request within ~30 seconds.
+# We set the timeout to 120s to be safe.
```

Comment:
Also, add we can add a reference to fastchat: https://github.com/lm-sys/FastChat/blob/main/fastchat/constants.py#L39

Reply:
Added. Thanks!